<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1036</width>
    <height>807</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Pruebas</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTreeWidget" name="listTestWidget">
          <column>
           <property name="text">
            <string/>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QFrame" name="frame_toolbar_video">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <layout class="QHBoxLayout" name="layout_toolbar_video">
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="btn_FullScreen">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>FullScreen</string>
             </property>
             <property name="shortcut">
              <string>F</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_refresh_vng">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Buscar VNG</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cb_resolution">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="currentText">
              <string/>
             </property>
             <property name="currentIndex">
              <number>-1</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButton">
             <property name="text">
              <string>...</string>
             </property>
             <property name="popupMode">
              <enum>QToolButton::ToolButtonPopupMode::InstantPopup</enum>
             </property>
             <property name="toolButtonStyle">
              <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QSlider" name="slider_erode_right">
            <property name="maximum">
             <number>10</number>
            </property>
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="slider_nose_width">
            <property name="maximum">
             <number>50</number>
            </property>
            <property name="singleStep">
             <number>0</number>
            </property>
            <property name="pageStep">
             <number>10</number>
            </property>
            <property name="value">
             <number>25</number>
            </property>
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="invertedAppearance">
             <bool>false</bool>
            </property>
            <property name="invertedControls">
             <bool>false</bool>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TickPosition::TicksBelow</enum>
            </property>
            <property name="tickInterval">
             <number>5</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="slider_erode_left">
            <property name="maximum">
             <number>10</number>
            </property>
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QSlider" name="slider_th_right">
            <property name="orientation">
             <enum>Qt::Orientation::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="CameraFrame">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="slider_th_left">
            <property name="orientation">
             <enum>Qt::Orientation::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Orientation::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QSlider" name="slider_vertical_cut_up">
              <property name="minimum">
               <number>-50</number>
              </property>
              <property name="maximum">
               <number>50</number>
              </property>
              <property name="tracking">
               <bool>true</bool>
              </property>
              <property name="orientation">
               <enum>Qt::Orientation::Vertical</enum>
              </property>
              <property name="invertedAppearance">
               <bool>true</bool>
              </property>
              <property name="tickPosition">
               <enum>QSlider::TickPosition::TicksBelow</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="slider_vertical_cut_down">
              <property name="minimum">
               <number>-50</number>
              </property>
              <property name="maximum">
               <number>50</number>
              </property>
              <property name="orientation">
               <enum>Qt::Orientation::Vertical</enum>
              </property>
              <property name="tickPosition">
               <enum>QSlider::TickPosition::TicksBelow</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="check_simultaneo">
              <property name="text">
               <string/>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Orientation::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btn_start">
            <property name="text">
             <string>Iniciar</string>
            </property>
            <property name="shortcut">
             <string>Space</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_fixed">
            <property name="locale">
             <locale language="English" country="UnitedStates"/>
            </property>
            <property name="text">
             <string>Fijar</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="lbl_test">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbl_time">
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="lbl_text_temp">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="layout_graph">
          <property name="topMargin">
           <number>30</number>
          </property>
         </layout>
        </item>
        <item>
         <widget class="QSlider" name="slider_time">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1036</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCalofrica">
    <property name="title">
     <string>VNG</string>
    </property>
    <widget class="QMenu" name="menuPruebas_Calor_cas">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Pruebas Calorícas</string>
     </property>
     <addaction name="actionOD_44"/>
     <addaction name="actionOI_44"/>
     <addaction name="actionOD_37"/>
     <addaction name="actionOI37"/>
    </widget>
    <addaction name="actionEspont_neo"/>
    <addaction name="menuPruebas_Calor_cas"/>
   </widget>
   <widget class="QMenu" name="menuOculomotoras">
    <property name="title">
     <string>Oculomotoras</string>
    </property>
    <addaction name="actionSeguimiento_Lento"/>
    <addaction name="actionOptoquinetico"/>
    <addaction name="actionSacadas"/>
   </widget>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionAbrir"/>
    <addaction name="actionNewUser"/>
    <addaction name="separator"/>
    <addaction name="actionTestNoSave"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuConfiguraci_n">
    <property name="title">
     <string>Configuración</string>
    </property>
    <addaction name="actionCambiar_evaluador"/>
   </widget>
   <widget class="QMenu" name="menuPosicionales">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Posicionales</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHerramientas">
    <property name="title">
     <string>Herramientas</string>
    </property>
    <addaction name="actionCalculadora_hipo_dp"/>
    <addaction name="separator"/>
    <addaction name="actionCalibrar"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuCalofrica"/>
   <addaction name="menuOculomotoras"/>
   <addaction name="menuPosicionales"/>
   <addaction name="menuHerramientas"/>
   <addaction name="menuConfiguraci_n"/>
  </widget>
  <action name="actionSalir">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
  <action name="actionCalorica">
   <property name="text">
    <string>Calorica</string>
   </property>
  </action>
  <action name="actionOculomotora">
   <property name="text">
    <string>Oculomotora</string>
   </property>
  </action>
  <action name="actionEspontaneo">
   <property name="text">
    <string>Espontaneo</string>
   </property>
  </action>
  <action name="actionPosicional">
   <property name="text">
    <string>Posicional</string>
   </property>
  </action>
  <action name="actionOD_44">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>OD 44</string>
   </property>
  </action>
  <action name="actionOI_44">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>OI 44</string>
   </property>
  </action>
  <action name="actionOD_37">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>OD 37</string>
   </property>
  </action>
  <action name="actionOI37">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>OI 37</string>
   </property>
  </action>
  <action name="actionEspont_neo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Espontáneo</string>
   </property>
  </action>
  <action name="actionSeguimiento_Lento">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Seguimiento Lento</string>
   </property>
  </action>
  <action name="actionOptoquinetico">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Optoquinetico</string>
   </property>
  </action>
  <action name="actionSacadas">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Sacadas</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
  <action name="actionNewUser">
   <property name="text">
    <string>Nuevo Usuario...</string>
   </property>
  </action>
  <action name="actionCalibrar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Calibrar</string>
   </property>
  </action>
  <action name="actionAbrir">
   <property name="text">
    <string>Abrir...</string>
   </property>
  </action>
  <action name="actionCambiar_evaluador">
   <property name="text">
    <string>Cambiar evaluador</string>
   </property>
  </action>
  <action name="actionTestNoSave">
   <property name="text">
    <string>Test de Seguimiento sin guardar</string>
   </property>
  </action>
  <action name="actionCalculadora_hipo_dp">
   <property name="text">
    <string>Calculadora hipo/dp</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
